### PR TITLE
Addressing review comments for BD and Coverity examples

### PR DESCRIPTION
**Pull request by spurohitsynopsys**

### CODE REVIEW

**Review by sig-tithi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -130,8 +134,9 @@ jobs:
           coverity_url: ${{ secrets.COVERITY_URL }}
           coverity_user: ${{ secrets.COVERITY_USER }}
           coverity_passphrase: ${{ secrets.COVERITY_PASSPHRASE }}
-          coverity_project_name: ${{ secrets.COVERITY_PROJECT_NAME }}
-          coverity_stream_name: ${{ github.event.repository.name }}
+          coverity_project_name: ${{ github.event.repository.name }}
+          coverity_stream_name: ${{ github.event.repository.name }}-${{ github.ref_name }}
+          coverity_policy_view: 'Outstanding Issues'
```
Comment (sig-tithi) [inline README.md:139]: are we keeping policy_view in example or as discussed as it's optional field we can remove it.

File: README.md [hunk 2]
```diff
@@ -130,8 +134,9 @@ jobs:
           coverity_url: ${{ secrets.COVERITY_URL }}
```
Comment (sig-tithi) [inline README.md:134]: url is referred from vars not secrets in chuck's example. just fyi

**Review by sig-tithi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (spurohitsynopsys) [reply README.md:131]: keeping it

File: README.md (hunk 2 above)
Comment (spurohitsynopsys) [reply README.md:126]: it's on users, they can configure accordingly

**Comment by sig-rnd-io-testuser** [comment]
Approved.

**Pull request merged**